cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "http://localhost:8080",
+  "projectId": "digs58"
```

Comment:
randomly generated?

Reply:
They assigned me that, I just created the project. We don't need it to run it, but just nice to have

Reply:
Not sure if u can see this https://dashboard.cypress.io/projects/digs58